### PR TITLE
crm dashboard view

### PR DESCRIPTION
**Pull request by RomainLibert**
Task : https://www.odoo.com/web#id=1831168&action=333&active_id=965&model=project.task&view_type=form&menu_id=4720

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/crm/views/crm_lead_views.xml [hunk 1]
```diff
@@ -613,6 +613,29 @@
             </field>
         </record>
 
+        <record model="ir.ui.view" id="crm_case_graph_view_oppor">
```
Comment (tde-banana-odoo) [inline addons/crm/views/crm_lead_views.xml]: Xml ids ! Go go go update them all !

File: addons/crm/views/crm_lead_views.xml [hunk 2]
```diff
@@ -836,7 +890,7 @@ for rec in records:
 
         <!-- 'My Pipeline' menu : Server action, act_window_views and act_windows -->
         <record model="ir.actions.act_window" id="crm_lead_opportunities">
-            <field name="name">Opportunities</field>
+            <field name="name">pet Opportunities</field>
```
Comment (tde-banana-odoo) [inline addons/crm/views/crm_lead_views.xml]: What are "pet" Opportunities :) ?

File: addons/crm/views/crm_lead_views.xml [hunk 3]
```diff
@@ -613,6 +613,29 @@
             </field>
         </record>
 
+        <record model="ir.ui.view" id="crm_case_graph_view_oppor">
+            <field name="name">crm.lead.graph.view</field>
```
Comment (tde-banana-odoo) [inline addons/crm/views/crm_lead_views.xml]: Names ! Go go go udpate them all !

**Comment by tde-banana-odoo** [comment]
I think there are some mismatch in the commits, like "pet opportunities" that appear in one commit and disappear in the other. Maybe recheck everything once to be sure.

**Comment by RomainLibert** [comment]
I squashed the commits and renamed the xml_ids and the views

File: addons/crm/report/crm_opportunity_report_views.xml [hunk 4]
```diff
@@ -52,35 +52,6 @@
             </field>
         </record>
 
-        <!-- Custom reports (aka filters) -->
-        <record id="filter_opportunity_opportunities_cohort" model="ir.filters">
```
Comment (tde-banana-odoo) [inline addons/crm/report/crm_opportunity_report_views.xml]: Don't forget to explain in the commit message those have been removed on purpose :) .

**Comment by RomainLibert** [comment]
Merged here : https://github.com/odoo/odoo/commit/629577ac86bcf4f5018a367945b1932206c1ff0c

**Pull request closed without merging**